internal/runners/initialize/init.go

```diff
@@ -281,30 +288,35 @@ func (r *Initialize) Run(params *RunParams) (rerr error) {
 		}
 	}
 
-	rti, commit, err := runtime.Solve(r.auth, r.out, r.analytics, proj, &commitID, target.TriggerInit, r.svcModel, r.config, runtime.OptNoIndent)
+	// Solve runtime
+	solveSpinner := output.StartSpinner(r.out, locale.T("progress_solve"), constants.TerminalAnimationInterval)
+	bpm := bpModel.NewBuildPlannerModel(r.auth)
+	commit, err := bpm.FetchCommit(commitID, r.prime.Project().Owner(), r.prime.Project().Name(), nil)
```

Comment:
Yeah this could be cleaned up more, but I think this is more appropriate for https://activestatef.atlassian.net/browse/DX-2929

Otherwise this PR will never land.